config/Dockerfiles/linchpin-tests.sh

```diff
@@ -8,6 +8,12 @@ mkdir -p ${base_dir}/${distro}_logs
 
 TEST_DIR="./config/Dockerfiles/tests.d"
 
+if [ -z "${PROVIDERS}" ]; then
+    PROVIDERS="dummy openstack"
+fi
```

Comment:
Now I'm confused.  Won't this conflict with the getTargetsToTest code in jenkins?  If anything I would think you would update that to getProvidersToTest and change the p_targets in JenkinsfileContainer to p_providers.

Reply:
This is one of the questions I had for you. I couldn't find where to update this in jenkins. I'll dig into that shortly and see if that helps. :)